### DISCUSSION TITLE
WordPress like admin panel

### DISCUSSION THREAD

**Discussion by momin-riyadh** [payloadcms/payload#815] [Q&A] [answered]
I'm a Web Front-End Developer who will be responsible for the complete lifecycle of scalable, secure, and well-designed software products from research and design to implementation. I’ll be working closely with designers, analysts, quality assurance, and other teams to translate each design into real-life products.
 
 I'm just exploring payload! Looking for something similar to or better than WP CMS and I'm determined to switch MERN stack.
 
 My question is: Am I implementing payload for all kinds of application admin panels like e-commerce, erp, etc?

**Comment by jmikrut** [comment, marked as answer]
Hey @momin-riyadh — pumped that you've found us!

Payload can work for any type of app admin panel you can think of. Ecommerce and ERP included.

To-date, a ton of very advanced apps have been built which are powered by Payload:

- Uber-like snow plowing service, and another lawncare equivalent is coming out soon
- Virtual events platform
- Video game backend
- Ecommerce sites
- Large, enterprise websites
- Small designer websites
- Assessment tool-style questionnaire which calculates scores and presents them to users
- Broadcast platform

Lots more. You can build anything! It's a perfect replacement for WP.

**Reply by momin-riyadh** [reply]
Good to know! Let's the explore begin. I will be happy to know contribute guidelines and willing to suggest a lot of feature and many more.....

Thanks Payload Teams
